ppfleetx/models/vision_model/vit/vit.py

```diff
@@ -205,6 +288,76 @@ def forward(self, x):
         x = self.head(x)
         return x
 
+    def state_dict(self,
+                   destination=None,
+                   include_sublayers=True,
+                   structured_name_prefix="",
+                   use_hook=True):
+        state_dict = super().state_dict(destination, include_sublayers,
+                                        structured_name_prefix, use_hook)
+        if self.use_fused_attn:
+            new_dict = []
+            poped_keys = []
+            for key, value in state_dict.items():
+                if r'qkv_' in key:
```

Comment:
Too many if-else branches. Try to write codes like:


```
replaced_dict = {
    "qkv_": "qkv.",
    "linear_": "proj.",
    ....
}

for key, value in state_dict.items():
    for rkey, rvalue in replaced_dict.items():
        if rkey in key:
            # write your codes here
```

Same below in `set_state_dict`.